### PR TITLE
Add Upgrade sandbox packages action

### PR DESCRIPTION
**Pull request by imsitnikov**

### CODE REVIEW

File: .github/workflows/upgrade-sandbox-packages.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - name: Upgrade sandbox packages
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          npm i @gravity-ui/uikit@latest @gravity-ui/components@latest @gravity-ui/date-components@latest @gravity-ui/icons
+          git config user.email ""
+          git config user.name "Upgrade sandbox packages action"
```
Comment (korvin89) [inline .github/workflows/upgrade-sandbox-packages.yml:27]: Do you want to shape message according to `commitlint` rules?

File: .github/workflows/upgrade-sandbox-packages.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          git checkout -b upgrade-sandbox-packages
+          git add package.json package-lock.json
+          export COMMIT_MESSAGE="Upgrade sandbox packages"
+          git commit -m "$COMMIT_MESSAGE"
+          git push --set-upstream origin upgrade-sandbox-packages --force
```
Comment (korvin89) [inline .github/workflows/upgrade-sandbox-packages.yml:32]: Do we need to add some unique id to the `upgrade-sandbox-packages` branch name?

Comment (imsitnikov) [reply .github/workflows/upgrade-sandbox-packages.yml:32]: For what reason? It is pushed with `force` flag

Comment (korvin89) [reply .github/workflows/upgrade-sandbox-packages.yml:32]: Possible collisions from previous attempts to update pkgs. But `force` should fix this problem

**Review by korvin89 — APPROVED** [review]

**Review by amje — APPROVED** [review]

File: .github/workflows/upgrade-sandbox-packages.yml (hunk 1 above)
Comment (imsitnikov) [reply .github/workflows/upgrade-sandbox-packages.yml:27]: Commit message changed to `chore(sandbox): upgrade sandbox packages`

**Pull request merged**